ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java

```diff
@@ -3154,4 +3154,20 @@
     @LogMessage(level = ERROR)
     @Message(id = 502, value = "Exception checking if timer %s should run")
     void exceptionCheckingIfTimerShouldRun(Timer timer, @Cause Exception e);
+
+    @LogMessage(level = WARN)
+    @Message(id = 503, value = "[EJB3.2 spec, section 5.6.4] Message Driven Bean 'onMessage' method %s can not be final.")
+    void mdbOnMessageMethodCantBeFinal(String className);
```

Comment:
Method parameter is a className but is printed out after method; not in sync with the rest of the messages.

Reply:
Oh good catch! I'll fix that and see if I can add both the className and the methodName. In case someone deploys several invalid Mdbs at the same time it might be nice to know what errors refers to what...